GetDKAN/dkan-tools
# Change behavior of restoreDb command and move namespaces around

Fixes #66 

This moves `dkan:restore` into its own class: `RestoreCommands` and consolidates some logic:

1. There is no longer a `-b` switch in the `install` command. 
2. The `dkan:restore` command is now simply `restore`.
3. A new command, `restore:db`, can take a URL argument, a filename inside _/backups, or will offer a selection of files inside _/backups_ if no argument given.
4. The `dktl make` command now makes DKAN and Drupal in one step. All make-related code is now in its own class.
5. `dktl dkan:deploy` is now simply `dktl deploy`. 

## QA Steps

1. Try to clone an existing site and adapt the _dktl.yml_ file to remove the `dkan:` namespace from the restore command. Try `dktl restore`; it should work as expected.
2. Create a local db dump and place it in _/backups_. Typing `dktl restore:db` should restore it.
3. Create a second db dump in _/backups_. Typing `dktl restore:db` should provide a selection. 
4. Try creating a site from scratch. Add some overrides to _drupal.make_ (as suggested in docs). Run `dktl make` and check behavior for any issues.
